waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -83,16 +104,16 @@ npmDepsForWasp =
           []
     }
 
-generateGitignore :: Generator FileDraft
-generateGitignore =
+genGitignore :: Generator FileDraft
+genGitignore =
```

Comment:
I noticed we were predominantly using `genFoo` instead of `generateFoo`, so I changed the ones I encountered to match this convention.

On a related note, there are a lot very similar functions between `WebAppGenerator` and `ServerGenerator`. How come they aren't extracted into common?

This is the list of shared functions. All of them differ in terms of `asWebAppFile` and `asServerFile`, some differ in other ways. However, I think they all essentially do the same thing and we could probably parameterize the differences:
```
genGitignore 
genNpmrc 
genNvmrc 
genPackageJson 
genReadme 
genSrcDir 
npmDepsForWasp 
```

Reply:
@Martinsos This is the duplication we mentioned on the weekly planning.

Reply:
Nice job renaming! Yes, later we started using `gen` but still had some `generate` left.

Regarding duplication -> You are right that we have some duplication, but I feel that probably it is not worth spending time on there right now, that we will overengineer. So I would leave it as it is for now, and let's see how painful it will be in the future.